watchos/CountlyTestApp-watchOS/AppDelegate.m

```diff
@@ -17,7 +17,7 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
     config.enableDebug = YES;
 
 //  config.deviceID = @"customDeviceID"                               //Optional custom or system generated device ID
-//  config.features = @[CLYPushNotifications, CLYCrashReporting, CLYViewTracking];     //Optional features
+//  config.features = @[CLYFeature.pushNotifications, CLYCrashReporting, CLYViewTracking];     //Optional features
```

Comment:
For Objective c code don't need to change it, it will not work for Objective C
Its only needed for Swift

Reply:
done